CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Java Access Bridge Wrapper changelog
 
+## 1.1.0 (date: 03.10.2023)
+
+- Add node/element refresh capability
+- Support for limiting the element tree maximum depth
+
+## 1.0.0 (date: 26.09.2023)
+
+- Regular expression support with elements search
+- Full project revamp and DX improvement
```

Comment:
The changelog for v1.0.0 could probably use some details on breaking changes.

Even thought this is an update from v0 to v1 which technically does not require such.. this library has been in use for years at this point so existing users need help if the interface has changed.